accounts/tests/tests.py

```diff
@@ -0,0 +1,23 @@
+from django.template.loader import get_template
+
+from house.constants import USER_LOGIN_PAGE_ROUTE, USER_SIGNUP_ROUTE
+
+
```

Comment:
Are these tests alligned with what Barak told us our tests were missing (in the Demo)?
about the templates, that we need to check if the right template is presented

Reply:
I think that it is.
@adiabramovitch what do you think about it?

Reply:
You are using the right object but you should check also the response URL to see that you have rendered the right template. 
for example: 
https://github.com/redhat-beyond/ResuMe/blob/08fc56c240518185ba5136f9f4583e01e538f8e0/users/test_users_views.py#L48
Also, I would check the object **response** to see which other information it holds so I can verify more information about the request. 

Also, you should check **bad login** and **bad signup** doesn't matter it is a Django framework as Barak explained last week/